src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationUnitTest.java

```diff
@@ -399,4 +399,27 @@ public void testGetField(final GencodeFuncotation gencodeFuncotation, final Stri
     public void testGetFieldFail(final GencodeFuncotation gencodeFuncotation, final String fieldName) {
         gencodeFuncotation.getField(fieldName);
     }
+
+    @Test
+    public void testVariantClassificationSeverityOverride() {
+
+        // Check that we start with the defaults:
+        for ( final GencodeFuncotation.VariantClassification vc : GencodeFuncotation.VariantClassification.values()) {
+            Assert.assertEquals(vc.getSeverity(), vc.getDefaultSeverity());
+        }
+
+        // Now set some values and make sure they stick:
+        for (int i = 98; i < 173; ++i) {
+            GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE.setSeverity(i);
+            Assert.assertEquals(GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE.getSeverity(), i);
+        }
+
+        // Now reset the values:
+        GencodeFuncotation.VariantClassification.resetSeveritiesToDefault();
+
+        // Check that the reset works:
+        for ( final GencodeFuncotation.VariantClassification vc : GencodeFuncotation.VariantClassification.values()) {
+            Assert.assertEquals(vc.getSeverity(), vc.getDefaultSeverity());
+        }
+    }
 }
```

Comment:
Can you add an end-to-end integration test as well that uses the new arg?

Reply:
Sure.

Reply:
Fixed!